liboptv/src/segmentation.c

```diff
@@ -93,7 +93,6 @@ int targ_rec (unsigned char *img, target_par *targ_par, int xmin,
       /* => local maximum, 'peak' */
           {
             yn=i;  xn=j;
-            xn = xn;
```

Comment:
Then why did it work so far? I'm wondering. Should there be a regression test for it?